src/aiida/cmdline/params/options/commands/setup.py

```diff
@@ -181,6 +181,16 @@ def get_quicksetup_password(ctx, param, value):
     cls=options.interactive.InteractiveOption,
 )
 
+SETUP_PROFILE_NAME = options.OverridableOption(
+    '-p',
+    '--profile-name',
```

Comment:
```suggestion
    '-p',
    '--profile-name',
    'profile',
```
By adding this, I think the name of the argument that will be passed to the commands function definition will be `profile` instead of `profile_name`. This is preferable because the `ProfileParamType` will convert the profile name to a `Profile` instance. Now the `verdi profile setup` commands has a lot of `profile_name.name` which looks odd. Witht he suggested change it becomes `profile.name` which is better I think

Reply:
Ah, that's a nice feature, and something I wondered about in the past, as well. I agree that it's nicer with using `profile` and added that in my recent commit.